### ISSUE TITLE
Add ctrl+c handler

### ISSUE THREAD

**Issue by shenki** [amboar/fpos#28]
When categorising and you want to stop there, I want to press ctrl+c to stop here and save the current state. When you did this in your techtalk it backtraced.

**Comment by amboar** [comment]
Yep, this is something that needs improvement. We should also come up with a strategy to handle partial categorisation in case people get bored and want to quit without losing work
